perf/benchmark/graph_plotter/example_plot/benchmark.csv

```diff
@@ -0,0 +1,79 @@
+StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_istio_proxy_fortioclient,cpu_mili_avg_istio_proxy_fortioserver,cpu_mili_avg_istio_proxy_istio-ingressgateway,mem_Mi_avg_istio_proxy_fortioclient,mem_Mi_avg_istio_proxy_fortioserver,mem_Mi_avg_istio_proxy_istio-ingressgateway
```

Comment:
This csv file is intentionally uploaded for making an example of how to use the graph plotter.